docroot/modules/custom/uiowa_profiles/src/Form/SettingsForm.php

```diff
@@ -226,6 +232,10 @@ public function buildForm(array $form, FormStateInterface $form_state) {
             ],
             '#attributes' => [
               'data-directory-index' => $key,
+              'id' => 'delete-profiles-instance-' . $key,
```

Comment:
This is causing the form to submit. I think we need to let Drupal set the ID as it includes a hash, I think to prevent CSRF.

Reply:
Good catch, fixed!